src/test/java/fr/esgi/al5_2/Tayarim/ProprietaireControllerTest.java

```diff
@@ -36,279 +37,349 @@
 import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
 
 import java.time.LocalDateTime;
+import java.util.AbstractMap;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'java.util.AbstractMap' import. Should be before 'org.springframework.test.web.servlet.result.MockMvcResultMatchers'.